public/static/app/perf/perf.js

```diff
@@ -560,6 +560,18 @@ mciModule.controller('PerfController', function PerfController(
     }
   }
 
+  $scope.isCanary = function(test) {
+    return !!test.match(/^(canary|fio|iperf|NetworkBandwidth).*$/);
```

Comment:
This should  probably be **/^(canary\_.*|fio\_.*|iperf.*|NetworkBandwidth).*$/** and it would be good if it could be shared between EVG-6095 as a constant.

Reply:
That regex doesn't match the test names, but I will make your suggestion from the EVG-6095 a common constant for excluding canary tests.